src/coreComponents/fileIO/silo/SiloFile.cpp

```diff
@@ -1331,8 +1331,8 @@ static std::vector< int > getSiloNodeOrdering( ElementType const elementType )
     case ElementType::Triangle:      return { 0, 1, 2 };
     case ElementType::Quadrilateral: return { 0, 1, 2, 3 }; // TODO check
     case ElementType::Polygon:       return { 0, 1, 2, 3, 4, 5, 6, 7, 8 }; // TODO
-    case ElementType::Tetrahedron:   return { 1, 0, 2, 3 };
-    case ElementType::Pyramid:       return { 0, 3, 2, 1, 4, 0, 0, 0 };
+    case ElementType::Tetrahedron:   return { 0, 1, 3, 2 };
+    case ElementType::Pyramid:       return { 0, 2, 3, 1, 4, 0, 0, 0 };
```

Comment:
This isn't the goal of your PR, but I'm curious if (and why) these extra 0s at the end are important. Does it have something to do with squashing nodes of a hex into pyramid shape, or or can they be safely removed?

Reply:
That is how we used to do it in geos.

Reply:
Removed extra 0s.